src/ser/error.rs

```diff
@@ -17,8 +17,12 @@ pub enum Error {
         key: Bson,
     },
 
-    /// A general error that ocurred during serialization.
-    /// See: https://docs.rs/serde/1.0.110/serde/ser/trait.Error.html#tymethod.custom
+    /// An invalid string was specified.
+    InvalidCString(String),
+
+    /// A general error that occurred during serialization.
+    /// See: <https://docs.rs/serde/1.0.110/serde/ser/trait.Error.html#tymethod.custom>
+    #[non_exhaustive]
```

Comment:
Adding `non_exhaustive` here would actually be a breaking change, so we'll have to omit it.